qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -589,22 +712,27 @@ def properties(
         """Return ``None``, simulators do not have backend properties."""
         return None
 
-    def run(
+    def run(    # type: ignore[override]
             self,
-            qobj: Union[QasmQobj, PulseQobj],
+            qobj: Union[QasmQobj, PulseQobj, QuantumCircuit, Schedule,
+                        List[Union[QuantumCircuit, Schedule]]],
             job_name: Optional[str] = None,
             job_share_level: Optional[str] = None,
             job_tags: Optional[List[str]] = None,
             validate_qobj: bool = False,
             backend_options: Optional[Dict] = None,
-            noise_model: Any = None
+            noise_model: Any = None,
+            **kwargs: Dict
     ) -> IBMQJob:
         """Run a Qobj asynchronously.
 
         Args:
-            qobj: The Qobj to be executed.
-            backend_options: Backend options.
-            noise_model: Noise model.
+            qobj: An individual or a
```

Comment:
We did a similar thing in the AQT provider, where it was requested that we make the change qobj -> circuit with a deprecation warning for the kwarg change.

Reply:
I kept `qobj` because I didn't want to break those doing `backend.run(qobj=qobj)`. Apparently terra has a nice little decorator to handle just this, so it's now `circuits` (plural as it takes multiple).